### PR TITLE
chore: run prettier when commiting

### PR DESCRIPTION
**Pull request by marionebl**
depends on #127

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +25,13 @@
   "bugs": {
     "url": "https://github.com/meetalva/alva/issues"
   },
+  "lint-staged": {
+    "*.{ts,tsx}": [
+      "prettier --write",
+      "tslint -c tsconfig.json --fix",
+      "git add"
```
Comment (LKuechler) [inline package.json]: What happens with partial commits? Would they not be added completely?

Comment (marionebl) [reply package.json]: That is true for commits that contain file chunks, don't know if this is common practice in the team?

Comment (LKuechler) [reply package.json]: Good question. I just remembered that we had that problem before in a different project.
But i think we can merge it.

Can you just rebase this branch? I am very sure that a lot of the commits are already in the master. This way it more clear what new stuff comes in

Comment (marionebl) [reply package.json]: Done

**Pull request merged**